pkg/capabilities/capabilities.go

```diff
@@ -18,6 +18,10 @@ var (
 
 	// ErrUnknownCapability is thrown when an unknown capability is processed.
 	ErrUnknownCapability = errors.New("unknown capability")
+
+	// ContainerImageLabels - label can indicate the required
+	// capabilities required by containers to run the container image.
+	ContainerImageLabels = []string{"io.containers.capabilities"}
```

Comment:
Out of curiosity: why not make it a string?

Reply:
I want to be future proof.  If we move this to a standard, the standard might want a different string, and we would need to support both.